packages/vite/src/node/config.ts

```diff
@@ -542,10 +542,13 @@ export interface ExperimentalOptions {
   /**
    * Enable builtin plugin that written by rust, which is faster than js plugin.
    *
+   * - 'resolver': Enable only the native resolver plugin.
+   * - 'v1': Enable the first stable set of native plugins (including resolver).
```

Comment:
```suggestion
   * - 'v1': Enable the first stable set of native plugins (including resolver).
   * - true: Enables all the stable native plugins (currently an alias of 'v1', it will map to a newer one in the future versions).
```